lib/stacks/dynamodb_stack.ts

```diff
@@ -0,0 +1,26 @@
+import { aws_dynamodb, App, RemovalPolicy, Stack, StackProps } from 'aws-cdk-lib';
+import { BillingMode } from 'aws-cdk-lib/aws-dynamodb';
+
+export class DynamoDBStack extends Stack {
+  constructor(parent: App, name: string) {
+    super(parent, name);
+
+    const pasteBinTableName = 'PasteBinMetaData';
```

Comment:
create a constant class for all the strings.

Reply:
We don't use the resource names elsewhere. So, I believe it would be better to keep them local to the place where the resources are being created.

Reply:
Next time when we use this elsewhere make sure we create a constant class as it would be easy to maintain.